desks-web-app/src/components/booking-components/WeekOverviewWindow.vue

```diff
@@ -1,20 +1,56 @@
 <template>
 
   <v-row>
-    <template v-for="n in 5" :key="n">
-      <v-col>
+    <template v-for="n in [0,1,2,3,4,5,6]" :key="n">
+      <v-col  :class="{days: true, monday: calculateNthNextDay(n).isoWeekday() === 1}" v-if="this.bookingDefaults && calculateNthNextDay(n).isoWeekday() <= 5">
+        <span id="nextWeek">Next Week</span>
         <availability-card
             v-if="this.bookingDefaults"
-            :startOfDay="calculateNthDayOfWeek(n)"
+            :startOfDay="calculateNthNextDay(n)"
             :room="this.bookingDefaults.room"
-            @add-booking="bookForDay(calculateNthDayOfWeek(n))"
+            @add-booking="bookForDay(calculateNthNextDay(n))"
         ></availability-card>
       </v-col>
     </template>
 
   </v-row>
 
 </template>
+<style>
```

Comment:
Do we really need customer styling Vuetify provides a bunch of components and prepared classes to style the component. The advantage of using the Vuetify styles and components is that they support the switch between light and darkmode and that they fit to the material design system.